contentcuration/contentcuration/static/js/handlebars/helpers.js

```diff
@@ -0,0 +1,49 @@
+var Handlebars = require("hbsfy/runtime");
+var marked = require("marked");
+marked.setOptions({
+  renderer: new marked.Renderer(),
+  gfm: true,
+  tables: false,
+  breaks: true,
+  pedantic: false,
+  sanitize: false,
+  smartLists: false,
+  smartypants: false
+});
+
+// A little bit of magic to let us use Django JS Reverse directly from inside a Handlebars template
+// Simply pass any arguments that you might otherwise use in order
+Handlebars.registerHelper('url', function(url_name) {
```

Comment:
We do, although it feels more like a gist, and after the horrendousness of our 'fle-utils' repo, and securesync repo, I am wary of trying to extract code too much.